deploy/lighthouse/crds/serviceexport_crd.yaml

```diff
@@ -0,0 +1,12 @@
+apiVersion: apiextensions.k8s.io/v1
```

Comment:
Need to use v1beta1. Refer: https://github.com/submariner-io/submariner-charts/pull/35

Reply:
Ack thanks